README.md

```diff
@@ -86,29 +112,29 @@ The `version` parameter processing is handled the same as `definition_table`.
-- fields: The list of defined fields
-    - name: The name of the field. This will be generated as a link based on `field_name_base_url`
-    - required: Whether or not the field is required. This is not necessarily whether the field is required by the FHIR<sup>®</sup> standard, but rather whether it is required by our server implementation.
-    - type: The type of the field. If found in `lib/resources/<version>/types.yaml`, this field will be linked to the specified resource.
-    - description: The description of the field.
-    - example: An example of how the field should be populated. The generated examples will be enclosed in &lt;pre&gt; tags to preserve formatting.
-    - note: Additional implementation notes.
-    - children: A list of nested fields. Each nested field item has the same structure as this `fields` list.
-    - url: Overrides the `field_name_base_url` generated URL if defined.
-    - binding: A list of terminology bindings supported for the field.
-      - description: Describes the purpose / intent of the binding.
-      - terminology: The list of terminologies supported by the field
-        - display: The terminology's display value.
-        - notes: Additional notes about the binding implementation.
-        - system: The actual system URI from which values may be returned.
-        - values: A list of individually supported values from the system.
+- field_name_base_url: `definition_table` will generate nested links for each field prepended with this url.
+- fields: The list of defined fields.
+  - name: The name of the field. This will be generated as a link based on `field_name_base_url`.
+  - required: Whether or not the field is required. This is not necessarily whether the field is required by the FHIR<sup>®</sup> standard, but rather whether it is required by our server implementation.
+  - type: The type of the field. If found in `lib/resources/<version>/types.yaml`, this field will be linked to the specified resource.
+  - description: The description of the field.
+  - example: An example of how the field should be populated. The generated examples will be enclosed in &lt;pre&gt; tags to preserve formatting.
+  - note: Additional implementation notes.
+  - children: A list of nested fields. Each nested field item has the same structure as this `fields` list.
+  - url: Overrides the `field_name_base_url` generated URL if defined.
+  - binding: A list of terminology bindings supported for the field.
+    - description: Describes the purpose / intent of the binding.
+    - terminology: The list of terminologies supported by the field.
+      - display: The terminology's display value.
+      - notes: Additional notes about the binding implementation.
+      - system: The actual system URI from which values may be returned.
+      - values: A list of individually supported values from the system.
 
 Standard YAML formatting rules apply
```

Comment:
Add '.' at the end.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/558/commits/ba15084162fd1b3fd3a969824d5dbfa87c79bffa#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R133